chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -648,5 +648,21 @@
                 </entry>
             </job-data-map>
         </job>
+        
```

Comment:
Our plan is to address these alerts, right? I don't see a reason to hold up the PR now, but maybe before it's merged?